source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```
@@ -1278,34 +1426,35 @@ a.initDockerSearch {
 
-input:hover[type=button] {
+input:hover[type="button"] {
   color: var(--unraid-66-color) !important;
   background: none !important;
   background-color: none !important;
 }
 
-input:hover[type=button][disabled] {
+input:hover[type="button"][disabled] {
   background: none !important;
   color: currentColor !important;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)